### PR TITLE
Minor updates to DR7 site to fix new issues since August.

### PR DESCRIPTION
**Pull request by geordie666**
This PR updates the DR7 part of the website to address issues that have been raised in the last month or so (https://github.com/legacysurvey/legacysurvey/issues/84).

### CODE REVIEW

File: pages/dr7/files.rst [hunk 1]
```diff
@@ -569,7 +569,7 @@ Name                                  Type         Units                 Descrip
 ``FIBERTOTFLUX_G``                    float32      nanomaggies           Predicted :math:`g`-band flux within a fiber from all sources at this location in 1 arcsec Gaussian seeing
 ``FIBERTOTFLUX_R``                    float32      nanomaggies           Predicted :math:`r`-band flux within a fiber from all sources at this location in 1 arcsec Gaussian seeing
 ``FIBERTOTFLUX_Z``                    float32      nanomaggies           Predicted :math:`z`-band flux within a fiber from all sources at this location in 1 arcsec Gaussian seeing
-``REF_ID``                            int64                              Reference catalog identifier for this star: "T2" for `Tycho-2`_, "G2" for `Gaia`_ DR2, empty otherwise
+``REF_ID``                            int64                              Reference catalog identifier for this star; Tyc1*1,000,000+Tyc2*10+Tyc3 for Tycho2; "sourceid" for Gaia-DR2
```
Comment (weaverba137) [inline pages/dr7/files.rst]: I would recommend making this an explicit `:math:` formula.

**Comment by weaverba137** [comment]
That's not quite what math mode is about.  You need to use full LaTeX notation:
```
:math:`\mathrm{Tyc1} \times 1,000,000 + \mathrm{Tyc2} \times 1,000 + \mathrm{Tyc3}`
```

**Comment by weaverba137** [comment]
OK, well, I guess I'll have to fix it in master.

**Comment by geordie666** [comment]
@weaverba137: apologies...the `:math:` version renders OK on GitHub, although I agree that your version looks better on the built website.

If you want to fix this in master, feel free. If you can tolerate it until the next set of issues arises, I'll open a new issue and fix it then.

There's no particular rush, but when you get a chance can you serve the new version of the website?

**Comment by weaverba137** [comment]
I'll be able to deploy shortly.

**Comment by weaverba137** [comment]
7.1.1 is deployed.

**Pull request merged**